src/main/frontend/commands.cljs

```diff
@@ -250,9 +251,14 @@
      ["Current time" #(date/get-current-time) "Insert current time"]
      ["Date picker" [[:editor/show-date-picker]] "Pick a date and insert here"]]
 
+    ;; order list
+    [["Number list" [[:editor/clear-current-slash]
+                     [:editor/toggle-own-number-list]] "Number list"]
+     ["Bullet list" [[:editor/clear-current-slash]
```

Comment:
Agree with your concerns! However, if the bullet list option is removed, the user can't return the default bullet list style directly through the slash command, even if it can be done by toggle number list or delete key.

Reply:
> the user can't return the default bullet list style directly through the slash command, even if it can be done by toggle number list

I'm confused by this sentence. By toggling with `/Number list`, the user can return to the default bullet list style. Is `/Bullet list`  just another way to do this? `/Bullet list` feels confusing because it doesn't do anything on a new block which is unlike all other commands

Reply:
In order to avoid confusion, I think the `/Bullet list` command can be removed at present. However, if the slash commands list can be prompted based on the block context in the future, the experience will be better :)